MainRepository/Program.cs

```diff
@@ -1,7 +1,44 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Implementation of Anagram
+class Anagram {
+
+   public static void Main (string[] args) {
+      Anagram a = new Anagram ();

```

Comment:
```suggestion
      Anagram a = new ();

```

Actually no need for a class for this assignment could have done in the main itself.